web-ui/app/locales/pt_BR/translation.json

```diff
@@ -82,7 +82,31 @@
         "tags": "Etiquetas"
     },
     "account-recovery": {
-      "page-title": "Pixelated Recuperação de Conta"
+      "page-title": "Pixelated Recuperação de Conta",
+      "admin-form": {
+        "image-description": "Código de Recuperação do Administrador - Passo 1 de 4",
+        "title": "Contate o administrador da sua conta e peça seu código",
+        "tip1": "A maneira mais segura é fazer isso pessoalmente.",
+        "tip2": "Você pode ligar ou mandar mensagem de texto se precisar.",
+        "tip3": "Nunca peça por email.",
+        "input-label": "digite aqui o código",
+        "button": "Próximo"
```

Comment:
Should we create just one key for the common words? We are repeating the button "Próximo" 3 times.